state-chain/cf-integration-tests/src/authorities.rs

```diff
@@ -503,10 +507,48 @@ fn can_move_through_multiple_epochs() {
 		.execute_with(|| {
 			let (mut testnet, _, _) = fund_authorities_and_join_auction(MAX_AUTHORITIES);
 			assert_eq!(GENESIS_EPOCH, Validator::epoch_index());
+			testnet.move_to_the_next_epoch();
+			witness_ethereum_rotation_broadcast(1);
 
-			for _ in 0..20 {
+			for i in 1..21 {
 				testnet.move_to_the_next_epoch();
+				witness_rotation_broadcasts([i + 1, i, i, i, i]);
```

Comment:
Not sure I get why Ethereum is always `+ 1` here. Is this because most keys are only created on the second rotation?

Reply:
yes, the first broadcast_id for eth is used in the first rotation where as other chains dont use the broadcast_id then (they are being activated manually) so in subsequent rotations, the broadcast id for other chains will be one less than ethereum.